app/assets/javascripts/components/composition-notes.jsx

```diff
@@ -0,0 +1,82 @@
+import PropTypes from 'prop-types'
+import Textarea from 'react-textarea-autosize'
+
+class CompositionNotes extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { notes: props.notes }
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({ notes: nextProps.notes })
+  }
+
+  onCompositionNotesChange(event) {
+    this.setState({ notes: event.target.value })
+  }
+
+  saveNotes(event) {
+    if (this.state.isRequestOut) {
+      return
+    }
+    if (event) {
+      event.target.blur()
+    }
```

Comment:
The blur is just to remove the border usually added to a clicked/focused button. So it goes from something like this:

<img width="116" alt="allow_saving_composition_notes_by_cheshire137_ _pull_request__136_ _cheshire137_overwatch-team-comps" src="https://user-images.githubusercontent.com/82317/27870234-f9bd0082-6167-11e7-8171-1d58d4351c0f.png">

To this:

<img width="115" alt="allow_saving_composition_notes_by_cheshire137_ _pull_request__136_ _cheshire137_overwatch-team-comps" src="https://user-images.githubusercontent.com/82317/27870250-04196598-6168-11e7-91f6-38cace3e65bc.png">

So it's appearance only, doing the blur.